src/layout/section/highlightSection/index.tsx

```diff
@@ -12,18 +13,16 @@ const StyledHighlightSection = styled(HighlightSection)`
   & .section-content {
     position: relative;
   }
+`
 
-  & .kirk-subheader {
-    color: ${color.white};
-    font-size: ${font.xl.size};
-    padding-top: ${space.xl};
-    padding-bottom: ${space.xl};
-  }
+export const StyledHighlightSectionSubHeader = styled(SubHeader)`
```

Comment:
I think you need to add a `&` to add specificity because `SubHeader` has one.

Reply:
I also had issues with it and never understood why it was here.

Reply:
I already tried, but still not working :(